connectors/src/api/webhooks/webhookSlack.ts

```diff
@@ -0,0 +1,131 @@
+import { Request, Response } from "express";
+
+import {
+  launchSlackSyncOneMessageWorkflow,
+  launchSlackSyncOneThreadWorkflow,
+} from "@connectors/connectors/slack/temporal/client";
+import { SlackConfiguration } from "@connectors/lib/models";
+import logger from "@connectors/logger/logger";
+import { withLogging } from "@connectors/logger/withlogging";
+import { ConnectorsAPIErrorResponse } from "@connectors/types/errors";
+
+type SlackWebhookReqBody = {
+  type?: string;
+  challenge?: string;
+  team_id?: string;
+  event?: {
+    channel?: string;
+    ts?: string; // slack message id
+    thread_ts?: string; // slack thread id
+    type?: string; // event type (eg: message)
+  };
+};
+
+type SlackWebhookResBody =
+  | { challenge: string }
+  | null
+  | ConnectorsAPIErrorResponse;
+
+const _webhookSlackAPIHandler = async (
+  req: Request<{}, SlackWebhookResBody, SlackWebhookReqBody>,
```

Comment:
SlackWebhookResBody in the request?

Reply:
yes, that's how it's defined:
```
// connectors/node_modules/@types/express/index.d.ts
    interface Request<
        P = core.ParamsDictionary,
        ResBody = any,
        ReqBody = any,
        ReqQuery = core.Query,
        Locals extends Record<string, any> = Record<string, any>
```

We do that in the other endpoints.